drivers/tee/tee_core.c

```diff
@@ -201,6 +268,29 @@ static int params_from_user(struct tee_context *ctx, struct tee_param *params,
 	return 0;
 }
 
+static void put_secure_memref(struct tee_shm *shm)
```

Comment:
move this to tee_shm.c: tee_shm.c may handle non-secure shm references and secure shm references.